x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/inline_tags_list.tsx

```diff
@@ -14,33 +14,39 @@ import { Query } from '../../types';
 
 interface Props {
   tags?: Query['tags'];
+  displayCountOnly?: boolean;
 }
-export const InlineTagsList: React.FC<Props> = ({ tags }) => {
+export const InlineTagsList: React.FC<Props> = ({ displayCountOnly, tags }) => {
   if (!tags?.length) return null;
 
   const displayedTags = tags.slice(0, 2);
-  const tooltipTags = tags.slice(2);
+  const tooltipTags = tags.slice(displayCountOnly ? 0 : 2);
+  const tagsText = tags.length === 1 ? 'tag' : 'tags';
+  const moreTagsMessage = displayCountOnly
+    ? `${tooltipTags.length} ${tagsText}`
+    : `and ${tooltipTags.length} more`;
 
-  return (
+  const TagToolTip = () => (
+    <EuiToolTip position="bottom" content={tooltipTags.join(', ')}>
+      <EuiBadge color={displayCountOnly ? 'hollow' : 'default'}>
+        {i18n.translate('xpack.enterpriseSearch.appSearch.engine.analytics.table.moreTagsBadge', {
+          defaultMessage: moreTagsMessage,
+          values: { moreTagsCount: tooltipTags.length },
+        })}
+      </EuiBadge>
```

Comment:
The i18n linter is complaining about this because this isn't quite the right way to handle i18n - we generally don't want conditionals inside i18n strings (which should be fairly static, except for any dynamic values), but conditionals should instead return individual i18n strings.

GitHub's being a butt about suggesting a diff for this entire blob here so I'm going to need to split this up into 2 comments:

```suggestion
      <EuiBadge color={displayCountOnly ? 'hollow' : 'default'}>{moreTagsMessage}</EuiBadge>
```

Reply:
https://github.com/elastic/kibana/pull/92939/commits/b3a8b4f9eaf881965defb7bdcfea01443adc9562